src/geometry/shape_views/collider_view.rs

```diff
@@ -245,166 +248,49 @@ impl<'a> ColliderView<'a> {
             #[cfg(feature = "dim3")]
-            ColliderView::RoundCylinder(c) => {
-                match c.raw.inner_shape.scaled(&scale.into(), num_subdivisions) {
-                    None => {
-                        log::error!("Failed to apply scale {} to RoundCylinder shape.", scale);
-                        SharedShape::ball(0.0)
-                    }
-                    Some(Either::Left(scaled)) => SharedShape::new(RoundShape {
-                        border_radius: c.raw.border_radius,
-                        inner_shape: scaled,
-                    }),
-                    Some(Either::Right(scaled)) => SharedShape::new(RoundShape {
-                        border_radius: c.raw.border_radius,
-                        inner_shape: scaled,
-                    }),
-                }
-            }
+            ColliderView::RoundCylinder(RoundCylinderView { raw }) => *raw,
             #[cfg(feature = "dim3")]
-            ColliderView::Cone(c) => match c.raw.scaled(&scale.into(), num_subdivisions) {
-                None => {
-                    log::error!("Failed to apply scale {} to Cone shape.", scale);
-                    SharedShape::ball(0.0)
-                }
-                Some(Either::Left(b)) => SharedShape::new(b),
-                Some(Either::Right(b)) => SharedShape::new(b),
-            },
+            ColliderView::RoundCone(RoundConeView { raw }) => *raw,
             #[cfg(feature = "dim3")]
-            ColliderView::RoundCone(c) => {
-                match c.raw.inner_shape.scaled(&scale.into(), num_subdivisions) {
-                    None => {
-                        log::error!("Failed to apply scale {} to RoundCone shape.", scale);
-                        SharedShape::ball(0.0)
-                    }
-                    Some(Either::Left(scaled)) => SharedShape::new(RoundShape {
-                        border_radius: c.raw.border_radius,
-                        inner_shape: scaled,
-                    }),
-                    Some(Either::Right(scaled)) => SharedShape::new(RoundShape {
-                        border_radius: c.raw.border_radius,
-                        inner_shape: scaled,
-                    }),
-                }
-            }
-            ColliderView::Compound(c) => {
-                let mut scaled = Vec::with_capacity(c.shapes().len());
-
-                for (tra, rot, shape) in c.shapes() {
-                    scaled.push((
-                        (tra * scale, rot).into(),
-                        shape.raw_scale_by(scale, num_subdivisions)?,
-                    ));
-                }
-                SharedShape::compound(scaled)
-            }
-        };
+            ColliderView::RoundConvexPolyhedron(RoundConvexPolyhedronView { raw }) => *raw,
+            ColliderView::Custom(raw) => *raw,
+        }
+    }
 
-        Some(result)
+    /// Compute the scaled version of `self.raw`.
```

Comment:
Maybe a more user-facing wording would be more appropriate ?

```suggestion
    /// Compute the scaled version of its inner [`Shape`].
```

Reply:
Sounds good